starlingcaptureapi/exif.py

```diff
@@ -56,6 +56,46 @@ def convert_timestamp(self, value_str):
         """
         return parser.parse(value_str).strftime("%Y:%m:%d %H:%M:%S %z")
 
+    def extract_from_exif_data(self, exif_data):
+        """Returns the latitude and longitude, if available, from the provided exif_data (obtained through get_exif_data above)"""
```

Comment:
This comment refers to "get_exif_data above", which is confusing, because there is no `get_exif_data` method in this file, or anywhere else in the codebase. The exif data seems to be coming from the PIL Image directly from the implementation of proofmode in `actions.py`.

Maybe update the comment to say something about the general shape of the `exif_data` that is expected.